### PR TITLE
Add Apollo backend

### PR DESCRIPTION
**Pull request by ArowShot**

### CODE REVIEW

File: kite-api/src/express.api.js [hunk 1]
```diff
@@ -2,9 +2,10 @@ require('dotenv').config();
 const express = require('express');
 const session = require('express-session');
 
+const { ApolloServer, gql } = require('apollo-server-express');
+
 const redis = require('redis').createClient(6379, 'kiteredis');
 const RedisStore = require('connect-redis')(session);
-const cors = require('cors');
```
Comment (ianfabs) [inline kite-api/src/express.api.js:7]: Why was CORS removed? Does it work without it now?

**Review by KyloRilo — APPROVED** [review]

File: kite-api/src/express.api.js (hunk 1 above)
Comment (ArowShot) [reply kite-api/src/express.api.js:7]: Apollo includes its own cors library

Comment (ianfabs) [reply kite-api/src/express.api.js:7]: 😎😎😎

**Review by ianfabs — APPROVED** [review]
😘

**Review by mattsre — APPROVED** [review]

**Pull request merged**